generate.xml

```diff
@@ -196,6 +204,12 @@
         <option value="--disable-tests" />
         <option value="--disable-samples" />
       </build>
+      <build name="png" version="1.6.21" parallel="true">
+        <option value="" />
+      </build>
+      <build name="qrencode" version="3.4.4" parallel="true">
+        <option value="" />
```

Comment:
Are these empty options necessary to satisfy `gsl.install.sh` or `install.sh`? If so it would be preferrable to modify the `gsl` to make this unnecessary.